tests/tests/test_github_webhooks.py

```diff
@@ -0,0 +1,605 @@
+        "debug:Trying to @mention the user in the newly created PR",
+        "debug:userName: tranchitella",
+        'github.CreateComment: org=mendersoftware,repo=workflows-enterprise,number=0,comment={"body":"@tranchitella I have created a PR for you, ready to merge as soon as tests are passed"}',
+        "info:Pull request event with action: closed",
+        "info:workflows:140 would trigger 0 builds",
+        "debug:Got push event :: repo workflows-enterprise :: ref refs/heads/master",
+        "git.Run: /usr/bin/git init .",
+        "git.Run: /usr/bin/git remote add github git@github.com:/mendersoftware/workflows-enterprise.git",
+        "git.Run: /usr/bin/git remote add gitlab git@gitlab.com:Northern.tech/Mender/workflows-enterprise",
+        "git.Run: /usr/bin/git fetch github",
+        "git.Run: /usr/bin/git checkout -b master github/master",
+        "git.Run: /usr/bin/git push -f gitlab master",
+        "info:Pushed ref to GitLab: workflows-enterprise:refs/heads/master",
+    ]
```

Comment:
Similarly here, this payload is being processed as a `pull_request` event (?) instead of a `push` event

Reply:
Found the issue: the test fixture was scope=session, and this made the logs to be retained across multiple tests, appending to the same list. It is now fixed.